core/controllers/admin_test.py

```diff
@@ -932,6 +932,7 @@ def test_changing_user_role_from_exploration_editor_to_topic_manager(self):
         topic_id = topic_services.get_new_topic_id()
         self.save_new_topic(
             topic_id, user_id, name='Name',
+            abbreviated_name='name', url_fragment='name',
```

Comment:
Try to use different args to symbolize different things -- e.g. 'abbrev', 'frag'

Reply:
Done.